src/xmipp/applications/scripts/deep_micrograph_cleaner/deep_micrograph_cleaner.py

```diff
@@ -72,30 +73,22 @@ def defineParams(self):
         self.addExampleLine('xmipp_deep_micrograph_cleaner -c path/to/inputCoords/ -o path/to/outputCoords -b $BOX_SIXE  -i  /path/to/micrographs/')
         
     def run(self):
-
-      try:
-        from xmippPyModules.deepLearningToolkitUtils.utils import checkIf_tf_keras_installed, updateEnviron
-      except ImportError:
-        print("Import error")
-        print(sys.path)
-        from pylib.xmippPyModules.deepLearningToolkitUtils.utils import checkIf_tf_keras_installed, updateEnviron
-
-        checkIf_tf_keras_installed()
+        # checkIf_tf_keras_installed()
         args={}
         gpusToUse="0"
         if self.checkParam('-g'):
           gpusToUse= self.getParam('-g')
           if "None" in gpusToUse or "-1" in gpusToUse:
```

Comment:
Maybe check if whatever is there is non-negative integer?

Reply:
it is a string. So you first should try to convert it

Reply:
Is it fine?